patch/etc/baseline/module-list.txt

```diff
@@ -25,6 +25,7 @@
 /com/fasterxml/jackson/dataformat/smile/main/jackson-dataformat-smile-2.8.9.jar
 /com/fasterxml/jackson/dataformat/xml/main/jackson-dataformat-xml-2.8.9.jar
 /com/fasterxml/jackson/dataformat/yaml/main/jackson-dataformat-yaml-2.8.9.jar
+/com/fasterxml/jackson/datatype/jackson-datatype-guava/main/jackson-datatype-guava-2.7.4.jar
```

Comment:
We need to keep all of the Jackson dependencies aligned with whatever version WildFly provides in the base layer (2.8.9). You'll need to `jackson-datatype-guava` to the BOM and then to `feature/modules/pom.xml`.

Reply:
Thanks for the info, will do.